deliver/lib/deliver/upload_assets.rb

```diff
@@ -1,6 +1,7 @@
 module Deliver
   class UploadAssets
     def upload(options)
+      return if options[:edit_live]
```

Comment:
No screenshots are being uploaded? If so, shouldn't we print out that this mode doesn't change the screenshots?